yarn.lock

```diff
@@ -6232,7 +6232,7 @@ matrix-encrypt-attachment@^1.0.3:
   resolved "https://registry.yarnpkg.com/matrix-encrypt-attachment/-/matrix-encrypt-attachment-1.0.3.tgz#6e016587728c396549c833985f39cbf6c07ee97b"
   integrity sha512-NwfoDY/yHL9Zo8KrY5GP8ymAoZJpEFKEK+IgJKdWf5xtsIFf7KIU2pbw8+Eq592j//nSDOC+/Ff4Fk8gVvDZpw==
 
-matrix-events-sdk@^0.0.1-beta.7:
+matrix-events-sdk@^0.0.1-beta.6, matrix-events-sdk@^0.0.1-beta.7:
```

Comment:
we should only be importing beta.7

Reply:
coming from old matrix-js-sdk in yarn cache maybe? fixed now